uber/utils.py

```diff
@@ -336,26 +340,46 @@ def to_dict(self):
         }
 
     @property
-    def models(self):
-        return self._models_cached
+    def is_empty(self):
```

Comment:
This new property name may be a bit misleading, since valid Charge() objects can exist without being tied to specific models (like charge objects for arbitrary charges). I'm not that concerned about it, but figured I should mention it.